pagerduty/komand_pagerduty/actions/get_on_call/action.py

```diff
@@ -50,9 +43,7 @@ async def async_get_users(self, user_ids: [str]) -> list:
                 tasks.append(
                     asyncio.ensure_future(connection.async_request(session=async_session, url=url, method="get"))
                 )
-                user_objects = await asyncio.gather(*tasks)
-                # extract "users" value from object
-                users = list()
-                for user in user_objects:
-                    users.append(user.get("user"))
-                return users
```

Comment:
I've also fixed some other code for this action, as there was a bug - the action was only returning 1 on-call user, rather than a list of on-call users.